### ISSUE TITLE
nil reference running FrameSort in Delves

### ISSUE THREAD

**Issue by rwygand** [Verubato/framesort#41]
```
553x FrameSort/Collections/Comparer.lua:176: attempt to compare number with nil
[string "@FrameSort/Collections/Comparer.lua"]:176: in function <FrameSort/Collections/Comparer.lua:119>
[string "=(tail call)"]: ?
[string "=(tail call)"]: ?
[string "=(tail call)"]: ?
[string "=[C]"]: in function `sort'
[string "@FrameSort/Modules/Sorting/SecureNoCombat.lua"]:396: in function <FrameSort/Modules/Sorting/SecureNoCombat.lua:371>
[string "@FrameSort/Modules/Sorting/SecureNoCombat.lua"]:577: in function <FrameSort/Modules/Sorting/SecureNoCombat.lua:537>
[string "=(tail call)"]: ?
[string "@FrameSort/Modules/Sorting/Sorting.lua"]:39: in function `Run'
[string "@FrameSort/Modules/Runner.lua"]:72: in function `callback'
[string "@FrameSort/Scheduling/Scheduler.lua"]:61: in function `RunWhenCombatEnds'
[string "@FrameSort/Modules/Runner.lua"]:67: in function `Run'
[string "@FrameSort/Modules/Runner.lua"]:52: in function <FrameSort/Modules/Runner.lua:47>

Locals:
leftToken = "party1"
rightToken = "player"
isArena = nil
leftRole = "HEALER"
rightRole = "NONE"
leftSpec = 0
rightSpec = 103
specOrdering = <table> {
 62 = 14
 1467 = 23
 63 = 15
 250 = 4
 251 = 32
 64 = 16
 1468 = 13
 253 = 25
 65 = 7
 255 = 27
 256 = 9
 257 = 10
 258 = 18
 259 = 34
 260 = 35
 261 = 36
 262 = 19
 581 = 6
 264 = 11
 265 = 20
 1473 = 24
 267 = 22
 268 = 5
 269 = 38
 270 = 12
 70 = 28
 102 = 17
 71 = 29
 103 = 31
 72 = 30
 104 = 3
 577 = 39
 73 = 2
 263 = 37
 105 = 8
 252 = 33
 66 = 1
 266 = 21
 254 = 26
}
roleOrdering = <table> {
 DAMAGER = 5
 TANK = 1
 HEALER = 2
}
leftValue = 2
rightValue = nil
(*temporary) = <table> {
 62 = 14
 1467 = 23
 63 = 15
 250 = 4
 251 = 32
 64 = 16
 1468 = 13
 253 = 25
 65 = 7
 255 = 27
 256 = 9
 257 = 10
 258 = 18
 259 = 34
 260 = 35
 261 = 36
 262 = 19
 581 = 6
 264 = 11
 265 = 20
 1473 = 24
 267 = 22
 268 = 5
 269 = 38
 270 = 12
 70 = 28
 102 = 17
 71 = 29
 103 = 31
 72 = 30
 104 = 3
 577 = 39
 73 = 2
 263 = 37
 105 = 8
 252 = 33
 66 = 1
 266 = 21
 254 = 26
}
(*temporary) = <table> {
 DAMAGER = 5
 TANK = 1
 HEALER = 2
}
(*temporary) = "attempt to compare number with nil"
wow = <table> {
 CompactArenaFrameMember1 = CompactArenaFrameMember1 {
 }
 MAX_RAID_MEMBERS = 40
 Enum = <table> {
 }
 CompactArenaFrameTitle = CompactArenaFrameTitle {
 }
 SettingsPanel = SettingsPanel {
 }
 C_PvP = <table> {
 }
 CompactArenaFrame = CompactArenaFrame {
 }
 EventRegistry = <table> {
 }
 PartyFrame = PartyFrame {
 }
 EditModeManagerFrame = EditModeManagerFrame {
 }
 Settings = <table> {
 }
 SlashCmdList = <table> {
 }
 C_Timer = <table> {
 }
 CompactPartyFrame = CompactPartyFrame {
 }
 MEMBERS_PER_RAID_GROUP = 5
 Events = <table> {
 }
 CompactRaidFrameContainer = CompactRaidFrameContainer {
 }
 UIParent = UIParent {
 }
}
lgist = <table> {
 debug = false
 state = <table> {
 }
 cache = <table> {
 }
 commScope = "INSTANCE_CHAT"
 static_cache = <table> {
 }
 events = <table> {
 }
 hooked = true
 frame = LibGroupInSpecT-1.1_Frame {
 }
}

```

**Comment by Verubato** [comment]
Thanks for the bug report, will fix it shortly.

**Comment by Verubato** [comment]
Fixed in 6.13.6

**Comment by rwygand** [comment]
Thanks!
